src/live_restore/live_restore_fs.c

```diff
@@ -919,6 +929,61 @@ __wti_live_restore_fs_fill_holes(WT_FILE_HANDLE *fh, WT_SESSION *wt_session)
     return (ret);
 }
 
+/*
+ * __wti_live_restore_cleanup_tombstones --
+ *     Remove all tombstone files from the database.
+ */
+int
+__wti_live_restore_cleanup_tombstones(WT_SESSION_IMPL *session)
+{
+    WT_DECL_RET;
+    WT_CONNECTION_IMPL *conn = S2C(session);
+    WTI_LIVE_RESTORE_FS *fs = (WTI_LIVE_RESTORE_FS *)conn->file_system;
+    WT_FILE_SYSTEM *os_fs = fs->os_file_system;
+    WT_SESSION *wt_session = (WT_SESSION *)session;
+
+    char **files;
+    char *logpath, *filepath;
+    uint32_t count = 0;
+    WT_DECL_ITEM(buf);
+
+    fs->finished = true;
+
+    /* Remove tombstones in the destination directory. */
+    WT_RET(os_fs->fs_directory_list(os_fs, wt_session, fs->destination.home, NULL, &files, &count));
+    for (uint32_t i = 0; i < count; i++) {
+        if (WT_SUFFIX_MATCH(files[i], WTI_LIVE_RESTORE_FS_TOMBSTONE_SUFFIX)) {
+            WT_ERR(
+              __live_restore_create_file_path(session, fs->destination.home, files[i], &filepath));
+            __wt_verbose_info(
+              session, WT_VERB_LIVE_RESTORE, "Removing tombstone file %s", filepath);
+            WT_ERR(os_fs->fs_remove(os_fs, wt_session, filepath, 0));
+        }
+    }
+    if (F_ISSET(&conn->log_mgr, WT_LOG_CONFIG_ENABLED)) {
+        WT_ERR(__wt_scr_alloc(session, 1024, &buf));
+
+        WT_ERR(os_fs->fs_directory_list_free(os_fs, wt_session, files, count));
+
+        /* Remove tombstones in the log path. */
+        WT_ERR(__live_restore_create_file_path(
+          session, fs->destination.home, (char *)conn->log_mgr.log_path, &logpath));
```

Comment:
api_data says `log_path` can be absolute:
```
            the name of a directory into which log files are written. The directory must already
            exist. If the value is not an absolute path, the path is relative to the database
            home (see @ref absolute_path for more information)'''),
```
We'll need a check for that

Reply:
Do we support absolute path log paths? I think that needs to be its own ticket.